src/meshcat/commands.py

```diff
@@ -19,6 +20,9 @@ def __init__(self, geometry_or_object, material=None, path=[]):
             if isinstance(material, PointsMaterial):
                 self.object = Points(geometry_or_object, material)
             else:
+                if texts is not None:
+                    material.map = TextTexture(texts)
+                    material.needsUpdate = True
```

Comment:
Do we still need this code?

Reply:
Right, they only serve to simplify the call a little, but it's not too verbose otherwise either. Removed them.